shared/transforms/xccdf-addremediations.xslt

```diff
@@ -61,6 +61,8 @@
   <xsl:variable name="ref_nist800_171" select="$rule/xccdf:reference[starts-with(@href, 'http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-171')]/text()"/>
   <xsl:variable name="ref_pci_dss" select="$rule/xccdf:reference[starts-with(@href, 'https://www.pcisecuritystandards.org/')]/text()"/>
   <xsl:variable name="ref_cjis" select="$rule/xccdf:reference[starts-with(@href, 'https://www.fbi.gov/file-repository/cjis-security-policy')]/text()"/>
+  <xsl:variable name="ref_disa_ossrg" select="$rule/xccdf:reference[starts-with(@href, 'http://iase.disa.mil/stigs/srgs/Pages/index.aspx')]/text()"/>
+  <xsl:variable name="ref_disa_stigid" select="$rule/xccdf:reference[starts-with(@href, 'http://iase.disa.mil/stigs/os/unix-linux/Pages/index.aspx')]/text()"/>
```

Comment:
Since these identifiers are RHT specific, can we point to their redhat page instead?

https://iase.disa.mil/stigs/os/unix-linux/Pages/red-hat.aspx

Reply:
Maybe but right now we use this for the reference URL, let's not change everything in a single PR. If this URL should change it should be done separately.